class UserNotFound(Exception):
    pass


class UserAlreadyExist(Exception):
    pass


class ValidationError(Exception):
    pass
